### PR TITLE
searcher_logic

### LINKED ISSUE #13: Implement logic for searcher
[Details](https://trello.com/c/pWXiV3z6/1-implement-logic-for-searcher)

### PR DESCRIPTION
**Pull request by Morphikss**
Working searching logic, but I'm not sure exactly if it's good enough.

### CODE REVIEW

File: src/views/UserList/components/UsersToolbar/UsersToolbar.js [hunk 1]
```diff
@@ -29,7 +29,7 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const UsersToolbar = props => {
-  const { className, ...rest } = props;
+  const { className,onSearchChange, ...rest } = props;
```
Comment (sebapaszynski) [inline src/views/UserList/components/UsersToolbar/UsersToolbar.js]: onSearchChange is missing in props validation(react/prop-types)

File: src/views/UserList/UserList.js [hunk 2]
```diff
@@ -17,15 +17,28 @@ const UserList = () => {
   const classes = useStyles();
 
   const [users] = useState(mockData);
+  const [searchResults, setSearchResults] = useState(mockData);
+
+  const onSearchChange = (event) => {
+    const inputValue = event.target.value.toLowerCase();
+    let results = users.filter(user=> {
```
Comment (sebapaszynski) [inline src/views/UserList/UserList.js]: Use const instead

File: src/views/UserList/components/UsersToolbar/UsersToolbar.js [hunk 3]
```diff
@@ -29,7 +29,7 @@ const useStyles = makeStyles(theme => ({
 }));
 
 const UsersToolbar = props => {
-  const { className, ...rest } = props;
```
Comment (Morphikss) [inline src/views/UserList/components/UsersToolbar/UsersToolbar.js:32]: removed

File: src/views/UserList/UserList.js [hunk 4]
```diff
@@ -17,15 +17,28 @@ const UserList = () => {
...
+      const fullName = user.firstName + " " + user.lastName;
+      return fullName.toLowerCase().includes(inputValue)
+    })
+  setSearchResults(results);
+  }
 
   return (
     <div className={classes.root}>
```
Comment (Morphikss) [inline src/views/UserList/UserList.js:32]: ??

**Review by Morphikss — COMMENTED** [review]
??

**Review by sebapaszynski — APPROVED** [review]

**Pull request merged**